### PR TITLE
Used sol6 code as latest Utils code base

### PR DESCRIPTION
**Pull request by ayobuenavista**
This PR is removing all previous versions of utility files.
leaving only latest version of utility which is the version should be used by any repository using this repo.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,9 +19,9 @@ We use `yarn` as the package manager. You may use `npm` and `npx` instead, but c
 
 ### Example Commands
 - `yarn test` (Runs all tests)
-- `yarn buidler test --no-compile ./test/sol6/utils5.js` (Test only utils5.js for sol6)
+- `yarn buidler test --no-compile ./test/utils.js` (Test only utils.js)
 or
-- `./test.sh -f ./test/sol6/utils5.js` (Test only utils5.js for sol6)
+- `./test.sh -f ./test/utils.js` (Test only utils.js)
 
 ## Coverage with `buidler-coverage`
```
Comment (ilanDoron) [inline README.md]: why do we need to write buidler-coverage?

we have some obligation to promote buidler?
or other reason?

**Review by ilanDoron — COMMENTED** [review]
OK
looks good.
very clean now.
And I see files are linted. which is nice

Thoughts:
1. We can probably turn back on the solidity version checker in solhint rules, since this repo expected to only have 1 version at time
2. I want to remove noModifiers files, since there is a better way to do it. so if we ever need to use it. should try the better way... (if u recall I showed how Bancor did it)
Let me bring it up in our next team meeting. see if anyone proposes against.
3. I understand utils fix will be done in next PR
right?

File: README.md (hunk 1 above)
Comment (ayobuenavista) [reply README.md]: Oh woops, removing this

**Comment by ayobuenavista** [comment]
> OK
> looks good.
> very clean now.
> And I see files are linted. which is nice
> 
> Thoughts:
> 
> 1. We can probably turn back on the solidity version checker in solhint rules, since this repo expected to only have 1 version at time
> 2. I want to remove noModifiers files, since there is a better way to do it. so if we ever need to use it. should try the better way... (if u recall I showed how Bancor did it)
>    Let me bring it up in our next team meeting. see if anyone proposes against.
> 3. I understand utils fix will be done in next PR
>    right?

1. Turned back on
2. Okay we can discuss this later meeting.
3. Yes.

**Review by ilanDoron — APPROVED** [review]

**Review by manhlx3006 — APPROVED** [review]

**Pull request merged**